pkg/sql/backfill.go

```diff
@@ -383,6 +383,9 @@ func (sc *SchemaChanger) truncateAndBackfillColumnsChunk(
 		p := &planner{
```

Comment:
I would recommend creating a new function `makePlannerForBackfill()` which would be reused in these three locations, so that we only have one to change should the initialization parameters need updating again.